scipy/stats/_continuous_distns.py

```diff
@@ -781,12 +787,39 @@ def _pdf(self, x, c, d):
         # burr.pdf(x, c, d) = c * d * x**(-c-1) * (1+x**(-c))**(-d-1)
         return c * d * (x**(-c - 1.0)) * ((1 + x**(-c))**(-d - 1.0))
 
+    def _logpdf(self, x, c, d):
+        return np.log(c) + np.log(d) + (-c - 1.0) * np.log(x) \
```

Comment:
Don't use a backslash for line continuation.  From [the section of PEP 8 on line length](https://www.python.org/dev/peps/pep-0008/#maximum-line-length):

> The preferred way of wrapping long lines is by using Python's implied line continuation inside parentheses, brackets and braces. Long lines can be broken over multiple lines by wrapping expressions in parentheses. These should be used in preference to using a backslash for line continuation.

Reply:
done, added parens around entire expression.